_emails/providence.md

```diff
@@ -0,0 +1,41 @@
+---
+title: Providence, RI
+permalink: "/providence"
+name: Letter to Governor, Mayor, and City Council
+city: Providence
+state: RI
+layout: email
+recipients:
+- governor@governor.ri.gov
+- Mayor@providenceri.gov
+- Ward2@providenceri.gov
+- Ward3@providenceri.gov
+- Ward4@providenceri.gov
+- Ward5@providenceri.gov
+- Ward6@providenceri.gov
+- Ward7@providenceri.gov
+- Ward8@providenceri.gov
+- Ward9@providenceri.gov
+- Ward10@providenceri.gov
+- Ward11@providenceri.gov
+- Ward12@providenceri.gov
+- Ward13@providenceri.gov
+- Ward14@providenceri.gov
+- Ward15@providenceri.gov
+subject: Defund the Police Department
+body: |
+  Hello,
+
+  My name is [NAME] and I am a resident of Providence. I am writing to demand that Governor Raimondo, Mayor Elorza, and the members of the City Council do everything in their power to adopt a budget that redirects all funds from the Police Department and towards community wellbeing, mental health and the needs of its residents. It has become clear that police forces are ineffective at keeping local communities safe, and that they perpetuate the oppression of residents who are people of color, undocumented, mentally ill, disabled, and LGBTQ.
+
+  While Providence has received a $700,000 grant to formulate a Behavioral Health Response Team, that amount is a mere drop in the bucket of the nearly $90 million budgeted for Police in 2021. The current budget proposal has allocated $154,450 on guns and ammunition, a 19% increase from 2020, which itself was a 44% increase from previous year. Not to mention the $963,626 that is allocated for uniforms and "wearing apparel,” presumably including more of the riot gear that is currently being used to suppress peaceful protests. Imagine how much those dollars could accomplish when used to fund low-income and public housing, education, resources for formerly incarcerated individuals, and public transportation, as Direct Action for Rights and Equality has demanded.
```

Comment:
Change to (edited sentence 2 based on comments on the issue):


While Providence has received a $700,000 grant to formulate a Behavioral Health Response Team, that amount is a mere drop in the bucket of the nearly $90 million budgeted for Police in 2021. The current budget proposal has allocated $154,450 on guns and ammunition, $1,461,620 for 50 new police recruits, not to mention $963,626 that is allocated for uniforms and "wearing apparel,” presumably including more of the riot gear that is currently being used to suppress peaceful protests. Imagine how much those dollars could accomplish when used to fund low-income and public housing, education, resources for formerly incarcerated individuals, and public transportation, as Direct Action for Rights and Equality has demanded.

Reply:
incorporated these changes